.github/workflows/release.yml

```diff
@@ -0,0 +1,69 @@
+name: Release
+
+on:
+  release:
+    types: [prereleased, released]
+
+jobs:
+  release-image-manifest:
+    name: Release image and upload manifests
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_IMAGE_NAME: newrelic/nri-ecs
```

Comment:
have we created already the new repo giving the access to the fsiops?

Reply:
yes that was already in place